core/app/c/(public)/[communitySlug]/public/Header.tsx

```diff
@@ -1,22 +1,11 @@
 import type { ReactNode } from "react";
 
-import dynamic from "next/dynamic";
-
 import { Button } from "ui/button";
 
+import { CopyCurrentUrlButton } from "~/app/components/CopyCurrentUrlButton";
 import Logo from "~/app/components/Logo";
 import { HEADER_HEIGHT } from "~/lib/ui";
 
-const CopyCurrentUrlButton = dynamic(
-	() =>
-		import("~/app/components/CopyCurrentUrlButton").then(
-			(module) => module.CopyCurrentUrlButton
-		),
-	{
-		ssr: false,
-	}
-);
-
```

Comment:
In next 15 you can no longer do `ssr: false` dynamic imports in Server Components, they have to be done in client components.
In cases where this happened, I have simply moved the dynamic imports to their component file, as the reason for doing dynamic imports in these cases was not for code-splitting, but to simply forego SSR (because we are accessing `window` directly)